katello/katello-backup

```diff
@@ -7,14 +7,17 @@ require "yaml"
 
 @options = {}
 @dir = nil
+@is_foreman_proxy_content = !`rpm -qa | grep foreman-proxy-content`.empty?
+@databases = ['pulp', 'postgres', 'mongodb']
```

Comment:
I get the variable `@databases` is a tad misleading, but it's less about categorizing the literal databases and more about categorizing the three areas we want to back up and make them easy to remove in certain scenarios. Maybe we could rename to `@backups`?